src/main/java/se/sundsvall/parkingpermit/Constants.java

```diff
@@ -10,7 +10,20 @@ private Constants() {}
 	public static final String PROCESS_KEY = "process-parking-permit"; // Must match ID of process defined in bpmn schema
 	public static final String TENANTID_TEMPLATE = "PARKING_PERMIT"; // Namespace where process is deployed, a.k.a tenant (must match setting in application.yaml)
 
-	public static final String UPDATE_AVAILABLE = "updateAvailable";
+	public static final String CAMUNDA_VARIABLE_CASE_NUMBER = "caseNumber";
+	public static final String CAMUNDA_VARIABLE_MESSAGE_ID = "messageId";
+	public static final String CAMUNDA_VARIABLE_UPDATE_AVAILABLE = "updateAvailable";
 	public static final VariableValueDto TRUE = new VariableValueDto().type(ValueType.BOOLEAN.getName()).value(true);
 	public static final VariableValueDto FALSE = new VariableValueDto().type(ValueType.BOOLEAN.getName()).value(false);
+
+	public static final String PHASE_ACTUALIZATION = "Aktualisering";
```

Comment:
Should these variables be named "ACTIVITY_ACTUALIZATION", "ACTIVITY_INVESTIGATION" and so on?
(since they describe "activities" in camunda?)

Reply:
Not sure, this is a state for the errand in case-data and not something that has to do with Camunda (more than having a sub process with the same name to in some manner try to connect the different acitivities to case-datas model). I'll will think about it until tomorrow when we can discuss it further.